pages/linux/xterm.md

```diff
@@ -0,0 +1,23 @@
+# xterm
+
+> The xterm command is a terminal emulator for the X Window System.
+
+- Open the terminal with a title of `Example`:
+
+`xterm -T {{Example}}`
+
+- Open the terminal in fullscreen mode:
+
+`xterm -fullscreen`
+
+- Open the terminal with a dark blue background and yellow font:
+
+`xterm -bg {{darkblue}} -fg {{yellow}}`
+
+- Open the terminal in screen position x=200,y=20 with 100 columns and 35 rows:
```

Comment:
How about including more common terms like "characters" and "lines" along with (or instead of) "rows" and "columns"?

As for the positions, if they are in pixels, it would be good to specify that in the description, for extra clarity.